src/components/graphs/bargraph/HorizontalBar.js

```diff
@@ -49,22 +49,22 @@ const Horizontalbar = () => {
 
   useEffect(() => {
     const emptyHandTotal = dataList.filter((x, index) => {
-      return x.force_rank === 'Rank 2 - Empty-hand';
+      return x.force_rank === 'Rank 2';
```

Comment:
So when I console log out dataList on the HorizontalBar.js component, the object that gets returned shows the properties are actually named in that longer way, so need to stay the same

force_rank: "Rank 4 - Chemical & Electric"

https://lambda-students.slack.com/archives/C02C8UXQZPY/p1630625436105200?thread_ts=1630625365.104500&cid=C02C8UXQZPY